### PR TITLE
Update email UG and standardize to using examples instead of example

### PR DESCRIPTION
**Pull request by enhao25**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -110,12 +110,12 @@ Format: `delete_student INDEX` <br>
 Examples:
 * `delete_student 2` deletes the 2nd student in the address book
 
-### Listing all students' emails: `emails`
-Displays concatenated string of students' emails, separated by `;`. Useful for sending mass emails to students.
+### Listing (filtered) students' emails: `emails`
```
Comment (chwoozy) [inline docs/UserGuide.md]: Hahaha, wait I placed them in brackets because I'm not sure which term we should go with 😄

**Review by chwoozy — APPROVED** [review]
Some changes, otherwise LGTM! 🤡

File: docs/UserGuide.md (hunk 1 above)
Comment (enhao25) [reply docs/UserGuide.md]: Haha LMAO, but I think its fine tou, it allows the reader to understand that its filtered, and can be the full list also.

Comment (samleewy) [reply docs/UserGuide.md]: I'm not sure if the bracket filtered feels abit out of place. Maybe something like `Listing students' emails based on current list` might be better? Hahaha, maybe someone else can phrase it better!

**Review by samleewy — COMMENTED** [review]
Some comments as well!
Also @enhao25 could you help to update the TOC for emails? HAHA realise it ain't there.

File: docs/UserGuide.md (hunk 1 above)
Comment (enhao25) [reply docs/UserGuide.md]: Alright, updated to Listing students' emails based on current list instead and also updated the TOC

File: docs/UserGuide.md [hunk 2]
```diff
@@ -13,6 +13,8 @@ It allows for faster and more effective student management.
   * [Listing all students: `list_student`](#listing-all-students-list_student)
   * [Locating student profile by name: `find_student`](#locating-student-profile-by-name-find_student)
   * [Adding a student: `add_student`](#adding-a-student-add_student)
+  * [Deleting a student: `delete_student`](#deleting-a-student-delete_student)
```
Comment (samleewy) [inline docs/UserGuide.md:16]: Nice catch on the missing `delete_student`!

**Review by samleewy — APPROVED** [review]
looks good! thanks! 🎉

**Pull request merged**